### PR TITLE
[docs] Travis + Saucelabs example

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
\cc @DevExpress/testcafe

### CODE REVIEW

File: examples/running-tests-using-travis-and-saucelabs/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: Can we follow README style that we have for https://github.com/DevExpress/testcafe/blob/master/examples/basic/README.md ?

File: examples/running-tests-using-travis-and-saucelabs/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+A set of sample files that help you to learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs (see [Running Tests Using Travis CI and Sauce Labs](http://devexpress.github.io/testcafe/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html)).
+
+The following files are included:
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: No need for included files description

File: examples/running-tests-using-travis-and-saucelabs/README.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+* *server.js* - runs an HTTP server for the application.
+* *index.html* - the application's main page.
+* *package.json* - the project description file. The package.json file's `dependencies` list already contains all needed packages to run the application, the `devDependencies` list contains TestCafe and Sauce Labs browser provider. The file also includes the `test` script that starts the application and runs TestCafe tests on a Sauce Labs browser.
+* *tests/index-test.js* - a simple TestCafe test.
+* *.travis.yml* - a Travis CI configuration file.
+
+## Running the example
+
+1. Create a GitHub repository and copy the sample files to it.
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: Just write how to checkout files for further steps just give a link to recipe. No need to duplicate info here

**Comment by inikulin** [comment]
\r-

File: examples/running-tests-using-travis-and-saucelabs/README.md [hunk 4]
```diff
@@ -1,20 +1,9 @@
...
-A set of sample files that help you to learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs (see [Running Tests Using Travis CI and Sauce Labs](http://devexpress.github.io/testcafe/documentation/recipes/running-tests-using-travis-ci-and-sauce-labs.html)).
-
-The following files are included:
-
-* *server.js* - runs an HTTP server for the application.
-* *index.html* - the application's main page.
-* *package.json* - the project description file. The package.json file's `dependencies` list already contains all needed packages to run the application, the `devDependencies` list contains TestCafe and Sauce Labs browser provider. The file also includes the `test` script that starts the application and runs TestCafe tests on a Sauce Labs browser.
-* *tests/index-test.js* - a simple TestCafe test.
-* *.travis.yml* - a Travis CI configuration file.
+*A set of sample files that help you to learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs.*
```
Comment (VasilyStrelyaev) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: > help you to

please, not again =)

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by VasilyStrelyaev** [comment]
lgtm

File: examples/running-tests-using-travis-and-saucelabs/README.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# Example: Running Tests Using Travis CI and Sauce Labs
+
+*A set of sample files that help you learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs.*
+
+## Running the example
+
+Create a GitHub repository and copy the sample files to it.
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: How I can copy files? You need to checkout files first and install dependencies, see https://github.com/DevExpress/testcafe/blob/master/examples/basic/README.md

Comment (MargaritaLoseva) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: @inikulin 
Ok, I'll write about cloning the repository, but I don't yet understand why we need to run `npm install` from _examples\running-tests-using-travis-and-saucelabs_? We aren't going to run this example locally. So, it is enough to perform the following steps:
1. Clone the TestCafe repository to your machine.
git clone https://github.com/DevExpress/testcafe.git
cd examples/running-tests-using-travis-and-saucelabs
npm install

2. Create a new GitHub repository and copy the sample files from the _examples/running-tests-using-travis-and-saucelabs_ to your repository.

3. Go through steps described in the topic.

Is it right?

Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/README.md]: Yes

**Comment by MargaritaLoseva** [comment]
fpr

File: examples/running-tests-using-travis-and-saucelabs/index.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<html>
+ <head>
+  <script type="text/javascript">
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/index.html]: Just testing new review system

File: examples/running-tests-using-travis-and-saucelabs/package.json [hunk 7]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "running-tests-using-travis-and-saucelabs",
+  "version": "1.0.0",
+  "description": "A set of sample files that help you to learn how to run TestCafe tests in the cloud using Travis CI and Sauce Labs.",
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/package.json]: Just some additional review comment

File: examples/running-tests-using-travis-and-saucelabs/index.html [hunk 8]
```diff
@@ -0,0 +1,13 @@
...
+  function change()
+  {
+  document.getElementById("click-here").value="Hello!";
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/index.html]: Weird formating

File: examples/running-tests-using-travis-and-saucelabs/package.json (hunk 7 above)
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/package.json]: Why it's a `set of samples files` when it's an `example`?

File: examples/running-tests-using-travis-and-saucelabs/server.js [hunk 9]
```diff
@@ -0,0 +1,6 @@
+const connect = require('connect');
+const serveStatic = require('serve-static');
```
Comment (inikulin) [inline examples/running-tests-using-travis-and-saucelabs/server.js]: Declarations should be aligned by `=`

**Comment by inikulin** [comment]
\cc @churkin @helen-dikareva @AndreyBelym

**Pull request merged**